src/gui/MainWindow.cpp

```diff
@@ -36,20 +36,26 @@ using namespace std;
 
 
 
-MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent), m_mouseAction{nullptr}
+MainWindow::MainWindow(QWidget* parent) : QMainWindow(parent), m_mouseAction{ nullptr }
 {
     loadTheme();
     prepareUi();
     prepareDockables();
-
-    documentArea->addTab(new HelpWidget(), "Quick Start");
-    if(QCoreApplication::arguments().length()>1){
+    helpWidgetInstance = new HelpWidget();
+    documentArea->addTab(helpWidgetInstance, "Quick Start"); // Modified this line
+    if (QCoreApplication::arguments().length() > 1) {
         openFile(QString(QCoreApplication::arguments().at(1)));
     }
     Globals::mainWindow = this;
-    
+
+    if (helpWidgetInstance) {
+        // Connect the signal to a lambda function for debugging
+        connect(helpWidgetInstance, &HelpWidget::ctrlNPressed, this, &MainWindow::newFile);
+        connect(helpWidgetInstance, &HelpWidget::ctrlOPressed, this, &MainWindow::openFileDialog);
+    }
```

Comment:
I want something you to move that `connect` inside `MainWindow` constructor to `HelpWidget` constructor. So that `HelpWidget` takes its parent widget as a parameter in its constructor, and the signal-slot connection is made within the `HelpWidget` itself. This makes `HelpWidget` more self-contained and easier to manage.